frontend/src/services/HttpRequest.js

```diff
@@ -35,6 +37,9 @@ async function makeHttpRequest(endpoint, method = "GET", jsonBody = {}) {
     const responseBody = await response.json();
     if (!responseBody.success) {
         Alert.error("Something went wrong!");
+        if (responseStatus === 401 || responseStatus === 403) {
+            window.location.replace('https://knolist.us.auth0.com/authorize?audience=knolist&response_type=token&client_id=pBu5uP4mKTQgBttTW13N0wCVgsx90KMi&redirect_uri=https://knolist-api.herokuapp.com/auth/callback')
```

Comment:
The only suggestion here is to make each part of this URL a variable that can be changed, so that it's easier to update it when necessary, then here you just "build" the string from these different parts. For example, you could have a `const auth0BaseUrl = "https://knolist.us.auth0.com"` and `const audience = "knolist"`, and then you build you string like `const loginUrl = auth0BaseUrl + "/authorize?audience=" + audience + ...`.
Basically, turn the parameters of this string into actual variables.

Reply:
Sounds good! I probably will end up scrapping this code anyway because the Auth0 React package provides a way to redirect to login without specifying a link at all, so probably will work with its SDK package for the most part